web/modules/custom/dbcdk_community/src/CommunityApiConfiguration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\CommunityApiConfiguration.
+ */
+
+namespace Drupal\dbcdk_community;
+
+use DBCDK\CommunityServices\Configuration as Configuration;
+
+/**
+ * Community API Configuration Class.
+ *
+ * This class contains the configuration needed for the ApiClient to
+ * communicate with the Community Service.
+ *
+ * @See DBCDK\CommunityServices\ApiClient
+ */
+class CommunityApiConfiguration extends Configuration {
+
+  /**
+   * Set configuration for the Community Service Api Client.
+   */
+  public function __construct() {
+    parent::__construct();
+
+    $config = \Drupal::config('dbcdk_community.settings');
+    // The Community Service prefixes each endpoint with a slash,
+    // so we make sure the host does not end with one.
+    $host = rtrim($config->get('community_service_url'), '/');
+    $this->setHost($host);
```

Comment:
Fair enough ^.^ - By enabling it as a checkbox in the admin form (at some point in time) under an `advanced settings` tab, then it would still be possible to override it in settings.php if the developer wished to do so. – But we can talk about that when the need arises.